imjoy/imjoyPluginEngine.py

```diff
@@ -177,8 +171,8 @@ def get_drives():
     if opt.token is None or opt.token == "":
         with open(os.path.join(WORKSPACE_DIR, ".token"), "r") as f:
             opt.token = f.read()
-except Exception as e:
-    pass
+except Exception:
+    logger.debug("Failed to generate token")
```

Comment:
I added debug log here instead of silent pass. Is this ok?

Reply:
sounds good. To be more precise, here we are trying to recover a previously generated token from a file.